### PR TITLE
Merge jdk8u352-b06 to 0.35

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

File: closed/openjdk-tag.gmk [hunk 1]
```diff
@@ -1 +1 @@
-OPENJDK_TAG := jdk8u352-b05
+OPENJDK_TAG := jdk8u352-b06
```
Comment (JasonFengJ9) [inline closed/openjdk-tag.gmk:1]: Are the following commits from running merging script locally, was there a manual change?
https://github.com/ibmruntimes/openj9-openjdk-jdk8/commit/a311eb43bfc32cc61d43babc6fb6ba513c0132f4
https://github.com/ibmruntimes/openj9-openjdk-jdk8/commit/a121e27ed8729b102c32abcf79ade09eeda21d44
https://github.com/ibmruntimes/openj9-openjdk-jdk8/commit/f012e250ee2a68faa963c895bb366d302d98f08d
https://github.com/ibmruntimes/openj9-openjdk-jdk8/commit/bf694dcf2605f90a557f4f240367fcd9145d998a

Comment (pshipton) [reply closed/openjdk-tag.gmk:1]: I created https://github.com/ibmruntimes/openj9-openjdk-jdk8/commit/a121e27ed8729b102c32abcf79ade09eeda21d44 and https://github.com/ibmruntimes/openj9-openjdk-jdk8/commit/f012e250ee2a68faa963c895bb366d302d98f08d in the staging branch, which promoted. The merging script created the others. The merging script updated to jdk8u362-b00, and it's done it again in the staging branch (which isn't promoted).

Comment (pshipton) [reply closed/openjdk-tag.gmk:1]: Actually I think it wasn't the merging script that created the merges, but the promotion build. Seems openj9 didn't get merged into staging, so the promotion merged staging to openj9.

**Review by JasonFengJ9 — APPROVED** [review]
LGTM

**Pull request merged**